mlir/lib/Conversion/GENToLLVM/GENToLLVM.cpp

```diff
@@ -259,9 +259,33 @@ struct ConvertGENToLLVM final
       signalPassFailure();
   }
 };
+} // namespace
+
+//===----------------------------------------------------------------------===//
+// ConvertToLLVMPatternInterface implementation
+//===----------------------------------------------------------------------===//
 
+namespace {
+/// Implement the interface to convert MemRef to LLVM.
```

Comment:
Comment needs to be updated.

Reply:
Best part's I copied from Arith. Arith, CF and Complex share the same comment :laug